Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+#!/usr/vin/env groovy
+
+pipeline {
+    agent { label 'executor-v2'}
+    triggers { cron('0 14 * * 1-5') }
+    options { timestamps() }
+
+    stages {
+        stage('Build Seal container image') {
+            steps {
+                sh './bin/build.sh'
+            }
+        }
+
+        stage('Run Seal bot') {
+            when {
+                branch 'master'
+            }
+            steps {
+                sh './bin/morning_seal.sh'
+            }
+        }
+    }
+
+    post {
+        failure {
+            slackSend(color: 'danger', message: 'FAILURE: could not run the Seal bot')
+        }
+    }
+}
```

Comment:
Will this mean that when we push a change to this project, Seal will run and notify?

Reply:
I am not sure about this X.X

My intention is to have it only build and tag the image on push, but run the bot on trigger for the master branch. I am not sure enough in my Jenkins-fu to guarantee that's what the current setup does 💫

Reply:
I really think we need to figure out how to do two things independently:

1) Build and push the Seal image
2) Pull and run the Seal container.

It might also be worth setting up a `staging` Slack account that a developer can use while developing without spamming the `#developement` channel.

Reply:
We do indeed have a #robot-test channel for that purpose.

I agree that those two things can be separate.